server/src/Action/HelloAction.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Action;
+
+use Psr\Container\ContainerInterface;
+use Psr\Log\LoggerInterface;
+use Slim\Http\Request;
+use Slim\Http\Response;
+
+class HelloAction {
+
+	/**
+	 * @var ContainerInterface
+	 */
+	private $container;
+
+	/**
+	 * @var LoggerInterface
+	 */
+	private $logger;
+
+	/**
+	 * HelloAction constructor.
+	 * @param ContainerInterface $container
+	 */
+	public function __construct( ContainerInterface $container ) {
+		$this->container = $container;
+		$this->logger = $this->container->get( 'logger' );
+	}
+
+	/**
+	 * @param Request $request
+	 * @param Response $response
+	 * @param array $args
+	 * @return Response
+	 */
+	public function __invoke( Request $request, Response $response, $args ) {
```

Comment:
Why not have a method (static?) `helloAction()` instead of using a magic method? I imagine you'd want multiple actions in a single controller? (if we had those)

Reply:
I don't think we need controllers. We are not doing CRUD and the API is very simple. Then again, like I said in the description, this is a generic skeleton. If we need to add more actions to the same resource we can change it